docs/architecture-and-design/block-skip-chain.md

```diff
@@ -17,9 +17,10 @@ If <img src="https://render.githubusercontent.com/render/math?math=BlockIndex \m
 ![Block Skip List](BlockSkipList.png)  
 Worth mentioning, that block with index <img src="https://render.githubusercontent.com/render/math?math=2^{i}" /> will contain _i_ hashes in its header.
 
+### Ledger connectivity proof example
 Validation algorithm just finds the shortest path from last block in ledger to block _i_ and from block _i_ to the genesis block. The result is list of `BlockHeader` containing each block in the path.
 - Let's mark blocks in the picture as 0,1,...,7,8
```

Comment:
The validation algorithm finds the the shortest path from the last block in the ledger to block _i_ and from block _i_ to the genesis block. The result is a list of `BlockHeader` containing each block in the path.